packages/app/ios/Orbit/Intents.intentdefinition

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Reading this XML doesn't make it clear how this is setup, here is the UI provided by Xcode.
<img width="1166" alt="Screen Shot 2022-05-08 at 5 46 31 PM" src="https://user-images.githubusercontent.com/8084674/167323192-d8744a65-d954-4473-829a-03f0a09e419b.png">

<img width="1168" alt="Screen Shot 2022-05-08 at 5 46 39 PM" src="https://user-images.githubusercontent.com/8084674/167323196-d871f624-520a-43d3-b922-e94a56f77414.png">